packages/contracts-bedrock/scripts/go-ffi/differential-testing.go

```diff
@@ -388,6 +397,37 @@ func DiffTestUtils() {
 
 		packed, err := decodedScalars.Pack(scalars.BaseFeeScalar, scalars.BlobBaseFeeScalar)
 		checkErr(err, "Error encoding output")
+		fmt.Print(hexutil.Encode(packed))
+	case "encodeGasPayingToken":
+		// Parse input arguments
+		token := common.HexToAddress(args[1])
+		decimals, ok := new(big.Int).SetString(args[2], 10)
+		checkOk(ok)
+		name := [32]byte(common.FromHex(args[3]))
```

Comment:
there is no need to do type casting here into `[32]byte`

Reply:
```
2024-05-22T13:11:36.078797Z ERROR cheatcodes: non-empty stderr input=["scripts/go-ffi/go-ffi", "diff", "encodeGasPayingToken", "0x000000000000000F1eE2Ca6D0D676834a03103cD", "27", "0x84d5178fc6bf72ad788aa6e59716ad00131500156637f94d89f13038e3e4e723", "0xb1a961402b23a87902c5ef36dc2e9b0e558961aedac78b02db6f60960b6cb7b4"] stderr="panic: Error encoding gas paying token: abi: cannot use slice as type array as argument\n\ngoroutine 1 [running]:\nmain.checkErr(...)\n\t/Users/diego/0xfuturistic/optimism/packages/contracts-bedrock/scripts/go-ffi/utils.go:30\nmain.DiffTestUtils()\n\t/Users/diego/0xfuturistic/optimism/packages/contracts-bedrock/scripts/go-ffi/differential-testing.go:422 +0x41e4\nmain.main()\n\t/Users/diego/0xfuturistic/optimism/packages/contracts-bedrock/scripts/go-ffi/bin.go:11 +0x70\n"
```

Reply:
Ah i misread, see [this](https://github.com/ethereum/go-ethereum/blob/7fd7c1f7dd9ba8d90399df2f080e4101ae37a255/common/types.go#L72) function, then you don't need to cast

Reply:
bump on this comment